pyiso/isone.py

```diff
@@ -188,3 +204,66 @@ def parse_json_load_data(self, data):
                 return data['FiveMinSystemLoads']['FiveMinSystemLoad']
         except (KeyError, TypeError):
             raise ValueError('Could not parse ISONE load data %s' % data)
+
+    def parse_json_lmp_data(self, data):
+        """
+        Pull approriate keys from json data set.
+        Raise ValueError if parser fails.
+        """
+        try:
+            if self.options.get('latest'):
+                return data['FiveMinLmp']
+            else:
+                return data['FiveMinLmps']['FiveMinLmp']
+        except (KeyError, TypeError):
+            raise ValueError('Could not parse ISONE lmp data %s' % data)
+
+    def get_lmp(self, zone_name, latest=True, start_at=False, end_at=False, **kwargs):
+        # set args
+        self.handle_options(data='lmp', latest=latest,
+                            start_at=start_at, end_at=end_at, **kwargs)
+
+        # get location id
+        try:
+            locationid = self.locations[zone_name.upper()]
+        except KeyError:
+            raise ValueError('No LMP data available for location %s' % zone_name)
+
+        # set up storage
+        raw_data = []
+        parsed_data = []
+
+        # collect raw data
+        for endpoint in self.request_endpoints(locationid):
+            # carry out request
+            data = self.fetch_data(endpoint, self.auth)
+
+            # pull out data
+            try:
+                raw_data += self.parse_json_lmp_data(data)
+            except ValueError as e:
+                logger.warn(e)
+                continue
+
+        # parse data
+        for raw_dp in raw_data:
+            # set up storage
+            parsed_dp = {}
+
+            # add values
+            parsed_dp['timestamp'] = self.utcify(raw_dp['BeginDate'])
```

Comment:
caiso get_lmp return a dict of prices, this returns a list of dicts.  Is one preferable?